grafana_import/grafana.py

```diff
@@ -66,18 +66,29 @@ class Grafana(object):
    dashboards: t.List[t.Any] = []
 
    #***********************************************
-   def __init__( *args, **kwargs ):
-      self = args[0]
+   def __init__(self, **kwargs ):
 
-      config = { }
-      config['protocol'] = kwargs.get('protocol', 'http')
-      config['host'] = kwargs.get('host', 'localhost')
-      config['port'] = kwargs.get('port', 3000)
-      config['token'] = kwargs.get('token', None)
-      if config['token'] is None:
-         raise GrafanaClient.GrafanaBadInputError('grafana token is not defined')
-
-      config['verify_ssl'] = kwargs.get('verify_ssl', True)
+      # Configure Grafana connectivity.
+      if "url" in kwargs:
+         self.grafana_api = GrafanaApi.GrafanaApi.from_url(kwargs["url"])
```

Comment:
Maybe it is better to use the `from_env` factory method right away ...
```python
GrafanaApi.from_env()
```

-- https://github.com/panodata/grafana-client/#authentication